cinema/services.py

```diff
@@ -0,0 +1,2 @@
+def params_to_int(query_string):
+    return [int(el) for el in query_string.split(",")]
```

Comment:
Consider adding error handling to the `params_to_int` function to manage cases where the query string contains non-integer values. You could use a try-except block to catch `ValueError` exceptions and handle them appropriately, such as by returning an error message or an empty list.